internal/scimserver/auth_handler.go

```diff
@@ -46,7 +45,7 @@ func (h *authHandler) withAuthContext(r *http.Request) (*http.Request, error) {
 			Detail: "Request unauthenticated with " + auth.AuthScheme,
 		}
 	}
-	md := metadata.Pairs(auth.HeaderAuthorization, fmt.Sprintf("%s %v", auth.AuthScheme, token))
```

Comment:
This was the actual bug. The auth scheme was already part of the token.

Reply:
ok